site/pages/competencies/how-to-use.md

```diff
@@ -7,7 +7,7 @@ layout: o-layout-docs
 
 # {{page.title}}
 
-Engineering competencies are used to inform conversations about career progression between an engineer and their line manager. They define what an engineer is expected to be doing at a particular level. They are not a checklist, but a way to indicate what areas they may need to improve in. We also expect engineers to be meeting competencies from the levels _before_ their current one.
+Engineering competencies are used to inform conversations about career progression between an engineer and their line manager. They define what an engineer is expected to be doing at a particular level. They are not a checklist, but a way to indicate what areas they may need to improve in.
```

Comment:
This line is just moved below into the section on "Levels", which I think flows better. I also made a small language change because this sentence seems to indicate that all competencies _must_ be met.